rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2469,7 +2469,21 @@ def test_upgrade_db_31_to_32(user_data_dir):  # pylint: disable=unused-argument
         ('1111112', 1595640208, 'external', 'ETH', 'USD', 'buy', '1.5541', '22.1', '3.4', None, None, None),  # noqa: E501
         ('1111113', 1595640208, 'external', 'ETH', 'USD', 'buy', '1.5541', '22.1', None, 'USD', None, None),  # noqa: E501
     ]
-
+    # Check that there are invalid pairs of (event_identifier, sequence_index)
+    base_entries_query = 'SELECT * from history_events WHERE event_identifier="KRAKEN-REMOTE-ID3"'
+    result = cursor.execute(base_entries_query).fetchall()
+    assert len(result) == 5
+    assert len([row[2] for row in result]) == 5
```

Comment:
Lol man wtf. I had to look at this twice to understand how you use it to check the invalid indices. I thought it was copy paste row with different assertion at first. Probably the worst way to do it performance wise but since it's just a test I am gonna overlook as it's short :)